README.md

```diff
@@ -12,7 +12,13 @@ Get notified when a DOM element enters or exits the viewport. A small (~1.9kb gz
 
 ## Installation
 
-Either download the [latest release](https://raw.githubusercontent.com/camwiegert/in-view/master/dist/in-view.min.js) and include it in your markup or install with [npm](http://npmjs.com/package/in-view):
+Either download the [latest release](https://raw.githubusercontent.com/camwiegert/in-view/master/dist/in-view.min.js) and include it in your markup:
+
+```html
+<script src="https://raw.githubusercontent.com/camwiegert/in-view/master/dist/in-view.min.js"></script>
```

Comment:
Hey, there are a couple of issues here:

- Linking content from `raw.githubusercontent.com` is discouraged, so I'd prefer to ask people to download the script.
- Content from `raw.githubusercontent.com` gets a `Content-Type: text/plain` header, which will break linked scripts in Chrome.

So, I'm not opposed to showing a script tag in the installation section, but we'd need another CDN. Maybe it's worth trying to get in-view onto cdnjs.

Or just: `/path/to/in-view.min.js`